doc/user/content/get-started/_index.md

```diff
@@ -35,8 +44,15 @@ to make a compromise between the freshness of the results, the cost of
 refreshing the view, and the complexity of the SQL statements you can use.
 
 In Materialize, you don't have to make such compromises. Materialize supports
-incrementally updated materialized views that are **always fresh**, even when
-using complex SQL statements, like multi-way joins with aggregations. How?
+incrementally updated view results that are **always fresh** (even when using
+complex SQL statements, like multi-way joins with aggregations) for *both*:
+
+- [Non-materialized views **with an
+index**](/concepts/views/#indexes-and-non-materialized-views) and
+
+- [Materialized views](/concepts/views/#materialized-views).
+
+How?
```

Comment:
Again just a suggestion, but I think this distinction is too granular for an overview of key features. The key feature is "incremental updates" (vs. traditional databases); whether they're achieved via indexed views or materialized views is an implementation detail.